example/e2ee.js

```diff
@@ -1,6 +1,9 @@
-import {SFrame}	from "../Client.js";
-import {Utils}	from "../lib/Utils.js";
+import {SFrame}	from "../index.js";
 
+const senderPasswordInput = document.getElementById('senderPassword');
+const receiverPasswordInput = document.getElementById('receiverPassword');
+const senderPasswordForm = document.getElementById('senderPasswordForm');
+const receiverPasswordForm = document.getElementById('receiverPasswordForm');
```

Comment:
Adding html stuff to the example so this can be tested

Reply:
Excellent! 💪